src/tests/tests_factcheck_endpoint.py

```
@@ -0,0 +1,36 @@
+        payload = {
+            "claim": "The Eiffel Tower is located in Berlin."
+        }
+        response = await ac.post("/factcheck", json=payload)
+        assert response.status_code == 200
+        data = response.json()
+        assert "verdict" in data
+        assert "evidence" in data
+        assert data["verdict"] in ["true", "false", "uncertain"]
+# tests/test_factcheck_endpoint.py
+
+import pytest
```

Ruff (0.11.9)

Module level import not at top of file

(E402)